src/omnisharp/engines/LspEngine.ts

```diff
@@ -95,25 +101,24 @@ export class LspEngine implements IEngine {
             middleware: {
                 async provideDefinition(document, position, token, next) {
                     const result = await next(document, position, token);
+                    // Needs Metadata document support
```

Comment:
And source generator document support, true for all the locations with this comment.